src/commands/Build/buildCommand.ts

```diff
@@ -1,12 +1,11 @@
-import { Command } from 'commander';
 import chalk from 'chalk';
 import validator from './validator';
 import { readFileSync } from 'jsonfile';
 import { existsSync } from 'fs';
 import { buildUsingWebpack, buildVanillaJS, buildPlugin } from './builder';
 
-const buildCommand = (program: Command) => {
-  program
+const buildCommand = (program: any) => {
+  return program
```

Comment:
Due to this change, what will be changed with the behavior?

And for ts, it's not good to use "any".
Is it no problem with it?

Reply:
The difference is that when call buildCommand() func, we can assign to variable to reuse program param in buildCommand func

I will change Command to commander.Command, because there's a bug when use Command:

![Screen Shot 2023-07-25 at 16 42 18](https://github.com/kintone-labs/kintone-cli/assets/117815925/042fc14c-2f37-4cb8-8b69-250a91ea1ef5)

Reply:
And this bug make us can't run: npm run build

Reply:
This is my resolve, Please have a check.
Thank you!
https://github.com/kintone-labs/kintone-cli/pull/118/files/8084b2d54c4c4942daa58c68591cef7144e2c7a5..1369b1a9b421b7aa4f371ae432bb11bcfa6804d1#diff-a4f691db069f66fae46d57dfa75429d03b45d4912e0825cb280fbb577d43a0ae

Reply:
Got it. Thanks for the fix.

Reply:
> Umm, it seems working fine with this. import { Command } from 'commander';

I use that import method, but it doesn't work: npm run build.
The reason in here:
https://github.com/tj/commander.js/issues/1465

Reply:
I got it. Thanks for the reference, too.

Reply:
Sorry for this inconvenient.
I used the wrong version of Commander (v8). 
I just updated the commander to new version (v11). It worked as expected.
I updated code in here:
https://github.com/kintone-labs/kintone-cli/pull/118/commits/b49db44d57bbaba871e4bb818bcd563185222560

Another command will not be affected

Reply:
Thank you for the modification!